app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en" class="govuk-template ">
+  <head>
+    <title><%= content_for(:page_title) %> - Publish teacher training courses - GOV.UK</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+    <%= tag :meta, name: "viewport", content: "width=device-width, initial-scale=1" %>
+    <%= tag :meta, name: "theme-color", content: "#0b0c0c" %>
+    <%= tag :meta, name: "ROBOTS", content: "NOINDEX, NOFOLLOW" %>
```

Comment:
Do we really want to block Google? It would seem like we would want our API docs easy to find no?

Reply:
I'm not sure. Can we defer this decision until later? This was just copied from Apply.

Reply:
I'm going to wait until I can chat to the Apply team about why they did this.